rca/project_styleguide/templates/patterns/atoms/image/image.html

```diff
@@ -1,7 +1,7 @@
 {% load wagtailimages_tags %}
 
-{% image item.image fill-100x37 as imageSmall %}
-{% image item.image fill-1134x595 as imageLarge %}
+{% image image fill-100x37 as imageSmall %}
```

Comment:
Can we not use camel case for variable names please?

Reply:
I'll alert FE of this.